src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -401,6 +401,13 @@ private function registerOAuthConfiguration(ContainerBuilder $container, array $
         $container->setParameter('api_platform.oauth.authorizationUrl', $config['oauth']['authorizationUrl']);
         $container->setParameter('api_platform.oauth.refreshUrl', $config['oauth']['refreshUrl']);
         $container->setParameter('api_platform.oauth.scopes', $config['oauth']['scopes']);
+
+        if ($container->hasDefinition('api_platform.swagger.action.ui')) {
+            $container->getDefinition('api_platform.swagger.action.ui')->setArgument('$oauthPkce', $config['oauth']['pkce']);
```

Comment:
If possible it's better to not use the `'$foo'` syntax as it involved reflection and slows down a bit the build process.